vars/Debian.yml

```diff
@@ -4,6 +4,8 @@ packages_base:
   - apt-transport-https
   - debian-archive-keyring
   - ca-certificates
+  - '{{ python_executable }}-pip'
```

Comment:
If we use `python_executable` variable in role we need to set the default value.

Reply:
Fixed